int16 right_third
int16 right_second
int16 right_first
int16 sentor
int16 left_first
int16 left_second
int16 left_third
